src/control/cad.js

```diff
@@ -1,9 +1,11 @@
+import { times, sortedIndexBy } from 'lodash';
```

Comment:
We prefer to not add lodash as library if you can do it without.  IS that possible ?

Reply:
It's fairly easy to do without lodash as mentioned in the original comment. As discussed in the original post, I used it primarily for the sortedIndexedBy optimization of the list build

Reply:
@megawac  please load only add the module you need in pakage.json:

lodash.sortedindexby